[do not merge] Slow sign in/all changes

rasom opened the pull request:
for testing purposes

rasom committed:
batch

rasom committed:
firts attempt

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/1/) | 2018-12-17 06:53:01 | ~13 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-063948-066cc9-pr.dmg) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/1/) | 2018-12-17 06:53:13 | ~13 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-063945-066cc9-pr.apk) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/1/) | 2018-12-17 06:54:14 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-063946-066cc9-e2e.app.zip) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/1/) | 2018-12-17 06:55:57 | ~16 min | `ios` | [:package: package](https://i.diawi.com/m5MewJ) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/1/) | 2018-12-17 06:56:12 | ~16 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-063949-066cc9-pr.exe) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/1/) | 2018-12-17 06:59:15 | ~19 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-063944-066cc9-e2e.apk) |
| :heavy_check_mark: | 066cc998 | [#1](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/1/) | 2018-12-17 07:02:46 | ~22 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-063947-066cc9-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/2/) | 2018-12-17 07:06:38 | ~13 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-065317-7182bd-pr.apk) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/2/) | 2018-12-17 07:06:40 | ~13 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-065311-7182bd-pr.dmg) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/2/) | 2018-12-17 07:09:33 | ~15 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-065423-7182bd-e2e.app.zip) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/2/) | 2018-12-17 07:10:23 | ~14 min | `ios` | [:package: package](https://i.diawi.com/JWVrRM) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/2/) | 2018-12-17 07:13:40 | ~14 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-065921-7182bd-e2e.apk) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/2/) | 2018-12-17 07:19:03 | ~22 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-065620-7182bd-pr.exe) |
| :heavy_check_mark: | 7182bda7 | [#2](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/2/) | 2018-12-17 07:24:36 | ~21 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-070256-7182bd-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/3/) | 2018-12-17 07:40:14 | ~11 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-072851-686d09-pr.dmg) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/3/) | 2018-12-17 07:42:57 | ~14 min | `ios` | [:package: package](https://i.diawi.com/Jm8YUy) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/3/) | 2018-12-17 07:43:03 | ~14 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-072848-686d09-pr.apk) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/3/) | 2018-12-17 07:43:36 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-072848-686d09-e2e.app.zip) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/3/) | 2018-12-17 07:45:53 | ~17 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-072852-686d09-pr.exe) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/3/) | 2018-12-17 07:46:56 | ~18 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-072847-686d09-e2e.apk) |
| :heavy_check_mark: | 686d0905 | [#3](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/3/) | 2018-12-17 07:47:34 | ~18 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-072850-686d09-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/6/) | 2018-12-17 19:39:53 | ~13 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-192613-3099c9-e2e.app.zip) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/6/) | 2018-12-17 19:40:24 | ~14 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181217-192611-3099c9-e2e.apk) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/6/) | 2018-12-17 19:40:40 | ~14 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-192616-3099c9-pr.dmg) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/6/) | 2018-12-17 19:42:47 | ~16 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-192612-3099c9-pr.apk) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/6/) | 2018-12-17 19:43:54 | ~17 min | `ios` | [:package: package](https://i.diawi.com/hLTNba) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/6/) | 2018-12-17 19:44:49 | ~18 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-192617-3099c9-pr.exe) |
| :heavy_check_mark: | 3099c912 | [#6](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/6/) | 2018-12-17 19:45:24 | ~19 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181217-192615-3099c9-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/7/) | 2018-12-18 10:32:52 | ~13 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-101857-a2de82-pr.dmg) |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/7/) | 2018-12-18 10:33:39 | ~14 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-101849-a2de82-pr.apk) |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/7/) | 2018-12-18 10:33:50 | ~15 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-101848-a2de82-e2e.apk) |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/7/) | 2018-12-18 10:35:42 | ~16 min | `ios` | [:package: package](https://i.diawi.com/3jvD4K) |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/7/) | 2018-12-18 10:35:54 | ~16 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-101858-a2de82-pr.exe) |
| :heavy_check_mark: | a2de8202 | [#7](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/7/) | 2018-12-18 10:46:07 | ~27 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-101856-a2de82-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/9/) | 2018-12-18 13:46:52 | ~13 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-133336-505487-e2e.apk) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/9/) | 2018-12-18 13:47:45 | ~14 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-133337-505487-pr.apk) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/9/) | 2018-12-18 13:48:10 | ~14 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-133340-505487-pr.dmg) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/9/) | 2018-12-18 13:48:16 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-133338-505487-e2e.app.zip) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/9/) | 2018-12-18 13:49:36 | ~15 min | `ios` | [:package: package](https://i.diawi.com/bRukfa) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/9/) | 2018-12-18 13:50:29 | ~16 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-133341-505487-pr.exe) |
| :heavy_check_mark: | 50548745 | [#9](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/9/) | 2018-12-18 13:53:01 | ~19 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-133340-505487-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/10/) | 2018-12-18 14:17:26 | ~12 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-140525-ff0201-pr.dmg) |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/10/) | 2018-12-18 14:20:48 | ~15 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-140522-ff0201-e2e.app.zip) |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/10/) | 2018-12-18 14:21:00 | ~15 min | `ios` | [:package: package](https://i.diawi.com/n6BFFX) |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/10/) | 2018-12-18 14:21:34 | ~16 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-140522-ff0201-pr.apk) |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/10/) | 2018-12-18 14:21:43 | ~16 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-140520-ff0201-e2e.apk) |
| :heavy_check_mark: | ff0201b3 | [#10](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/10/) | 2018-12-18 14:30:20 | ~24 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-140524-ff0201-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 2029464e | [#11](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/11/) | 2018-12-18 19:45:59 | ~12 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-193329-202946-pr.dmg) |
| :heavy_check_mark: | 2029464e | [#11](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/11/) | 2018-12-18 19:47:14 | ~13 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181218-193323-202946-e2e.apk) |
| :heavy_check_mark: | 2029464e | [#11](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/11/) | 2018-12-18 19:47:20 | ~13 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181218-193325-202946-pr.apk) |
| :heavy_check_mark: | 2029464e | [#11](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/11/) | 2018-12-18 19:48:02 | ~14 min | `ios` | [:package: package](https://i.diawi.com/sqhLgN) |
| | | | | | | |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/13/) | 2018-12-19 07:36:07 | ~11 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181219-072450-c331c8-pr.dmg) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/13/) | 2018-12-19 07:39:35 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181219-072447-c331c8-e2e.app.zip) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/13/) | 2018-12-19 07:39:37 | ~14 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181219-072447-c331c8-pr.apk) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/13/) | 2018-12-19 07:39:47 | ~15 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181219-072445-c331c8-e2e.apk) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/13/) | 2018-12-19 07:40:31 | ~15 min | `ios` | [:package: package](https://i.diawi.com/V66jQj) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/13/) | 2018-12-19 07:44:36 | ~19 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181219-072449-c331c8-pr.AppImage) |
| :heavy_check_mark: | c331c8dc | [#13](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/13/) | 2018-12-19 07:47:07 | ~22 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181219-072451-c331c8-pr.exe) |
| | | | | | | |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/14/) | 2018-12-21 08:07:59 | ~7 min 0 sec | `windows` | [:package: package]() |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/14/) | 2018-12-21 08:13:11 | ~12 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181221-080057-ded458-pr.dmg) |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/14/) | 2018-12-21 08:15:51 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181221-080055-ded458-e2e.app.zip) |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/14/) | 2018-12-21 08:16:09 | ~15 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181221-080054-ded458-pr.apk) |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/14/) | 2018-12-21 08:16:19 | ~15 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181221-080053-ded458-e2e.apk) |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/14/) | 2018-12-21 08:18:48 | ~17 min | `ios` | [:package: package](https://i.diawi.com/ryovzV) |
| :heavy_check_mark: | ded45881 | [#14](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/14/) | 2018-12-21 08:20:54 | ~19 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181221-080057-ded458-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7110/15/) | 2018-12-25 19:28:02 | ~13 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181225-191453-bcf8ba-pr.apk) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7110/15/) | 2018-12-25 19:28:07 | ~13 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181225-191452-bcf8ba-e2e.apk) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7110/15/) | 2018-12-25 19:28:13 | ~13 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181225-191456-bcf8ba-pr.dmg) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7110/15/) | 2018-12-25 19:29:07 | ~14 min | `ios-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-181225-191453-bcf8ba-e2e.app.zip) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7110/15/) | 2018-12-25 19:30:31 | ~15 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181225-191456-bcf8ba-pr.exe) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7110/15/) | 2018-12-25 19:32:04 | ~17 min | `ios` | [:package: package](https://i.diawi.com/2mmBSS) |
| :heavy_check_mark: | bcf8bab4 | [#15](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7110/15/) | 2018-12-25 19:39:34 | ~24 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181225-191455-bcf8ba-pr.AppImage) |

rasom commented:
**galaxy s9, 200 chats, 50 messages in each:**
`21:53:15.945` :accounts.login.ui/password-input-submitted
`21:53:16.120` :START-NODE
`21:53:16.301` :init-chats
`21:53:16.771` :RENDERING-home
`21:53:18.473` :NODE-READY
`21:53:20.620` :accounts.login.callback/login-success
`21:53:21.067` :init-whisper
`21:53:21.451`:shh.callback/filters-added

UI: **0.83s**
Node starting: **2.35s**
Statusgo.Login: **2.15s**
Total: **5.5s**

**galaxy s9, 400 chats, 50 messages in each:**
`22:15:51.960` :accounts.login.ui/password-input-submitted
`22:15:52.340` :START-NODE
`22:15:52.591` :init-chats
`22:15:53.248` :RENDERING-home
`22:15:54.773` :NODE-READY
`22:15:57.006` :accounts.login.callback/login-success
`22:15:57.778` :init-whisper
`22:15:58.416` :shh.callback/filters-added

UI: **1.28s**
Node starting: **2.43s**
Statusgo.Login: **2.23s**
Total: **6.45s**

rasom commented:
**slow android device, no chats:**
`11:31:29.350` :accounts.login.ui/password-input-submitted
`11:31:29.360` "initializing realm encryption key..."
`11:31:29.380` "changing account to:  .." **(0.02s for getting key form keystore)**
`11:31:29.490` :init.callback/account-change-success **(0.13s for opening realm db)**
`11:31:29.540` :START-NODE
`11:31:30.000` :init-chats
`11:31:30.050` :RENDERING-home
`11:31:31.880` :NODE-READY
`11:31:40.360` :accounts.login.callback/login-success
`11:31:40.370` :init-whisper
`11:31:41.530` :shh.callback/filter-added

UI: **0.7s**
Node starting: **2.34s**
Statusgo.Login: **8.48s**
Total: **12.18s** (88% status-go side)

**slow android device, 20 chats, 50 messages in each:**
`10:35:09.260` :accounts.login.ui/password-input-submitted
`10:35:09.510` :START-NODE
`10:35:09.840` :init-chats
`10:35:10.060` :RENDERING-home
`10:35:11.930` :NODE-READY
`10:35:20.520` :accounts.login.callback/login-success
`10:35:20.700` :init-whisper
`10:35:22.750` :shh.callback/filters-added

UI: **0.8s**
Node starting: **2.42s**
Statusgo.Login: **8.6s**
Total: **13.5s**

**slow android device, 50 chats, 50 messages in each:**
`10:43:53.070` :accounts.login.ui/password-input-submitted
`10:43:53.080` "initializing realm encryption key..."
`10:43:53.380` "changing account to:  .." **(0.3s for getting key form keystore)**
`10:43:53.570` :init.callback/account-change-success **(0.5s for opening realm db)**
`10:43:53.670` :START-NODE
`10:43:54.170` :init-chats
`10:43:54.660` :RENDERING-home
`10:43:56.340` :NODE-READY
`10:44:05.420` :accounts.login.callback/login-success
`10:44:05.760` :init-whisper
`10:44:07.990` :shh.callback/filters-added

UI: **1.6s**
Node starting: **2.67s**
Statusgo.Login: **9.08s**
Total: **14.92s**

**slow android device, 100 chats, 50 messages in each:**
`11:13:02.660` :accounts.login.ui/password-input-submitted
`11:13:02.670` "initializing realm encryption key..."
`11:13:02.700` "changing account to:  .." **(0.03s for getting key form keystore)**
`11:13:02.850` :init.callback/account-change-success **(0.19s for opening realm db)**
`11:13:02.890` :START-NODE
`11:13:03.210` :init-chats
`11:13:03.860` :RENDERING-home
`11:13:05.600` :NODE-READY
`11:13:15.210` :accounts.login.callback/login-success
`11:13:15.860` :init-whisper
`11:13:17.980` :shh.callback/filters-added

UI: **1.2s**
Node starting: **2.71s**
Statusgo.Login: **9.61s**
Total: **15.32s**

rasom committed:
chats pagination

rasom commented:
**galaxy s9, 400 chats, 50 messages in each:** (at first last 15 chats are loaded, then the rest)

`18:41:36.722` :accounts.login.ui/password-input-submitted
`18:41:36.733` "initializing realm encryption key..."
`18:41:37.056` "changing account to:  .." **(0.323s for getting key form keystore)**
`18:41:37.194` :init.callback/account-change-success **(0.472s for opening realm db)**
`18:41:37.296` :START-NODE
`18:41:37.654` :home-did-mount
`18:41:37.810` :init-rest-of-chats
`18:41:39.513` :NODE-READY
`18:41:42.297` :accounts.login.callback/login-success
`18:41:42.302` :init-whisper
`18:41:42.875` :shh.callback/filters-added

UI: **0.932s**
Node starting: **2.217s**
Statusgo.Login: **2.784s**
Total: **6.153s**

**note**
"initializing realm encryption key..." is almost instant if logout/login again, thus UI time is slightly better - **0.488s**

rasom committed:
[slow sign in] Fetch generic password only once

Previously it was fetched each time when any realm db was opened,
but it should happen only once. Saves up to 300ms on sign in.

rasom commented:
on desktop (macos) node starting takes 2.133s, which relatively the same as time as on any other device

rasom committed:
log :home update

rasom commented:
### samsung a500h 
 30 chats, 50 messages in each, all not seen
 
`0.9.31`(empty)
22.73s (3.45s)

`0.9.32`(empty)
23.31s (11.98s)

latest(empty) the same time for 30 and 100 chats
2.33s (1.26s)

### samsung galaxy s9 
100 chats, 50 messages, all not seen

`0.9.31`(empty)
11.32s (0.9s)

`0.9.32`(empty)
11.86 (2.91s)

latest(empty) the same time for 100 and 200 chats
0.72s (0.18s)
